### ISSUE TITLE
Clean output if json parameter is not set

### ISSUE THREAD

**Issue by gavignon** [scolladon/sfdx-git-delta#378]
### Clean output if json parameter is not set
Remove json output if --json parameter is not set, use spinner instead.

### What is the current result ?
`sfdx sgd:source:delta -s force-app -f <FROM> -t <TO> -o <OUTPUT>` 
**Result:**
`{
  "error": null,
  "output": "<OUTPUT>",
  "success": true,
  "warnings": []
}`

---

### What are the expected results ?

1. Without JSON parameter:
`sfdx sgd:source:delta -s force-app -f <FROM> -t <TO> -o <OUTPUT>` 
**Result:**
`Generating delta...done` or **nothing**

2. With JSON parameter
`sfdx sgd:source:delta -s force-app -f <FROM> -t <TO> -o <OUTPUT> --json` 
**Result:**
`{
  "error": null,
  "output": "<OUTPUT>",
  "success": true,
  "warnings": []
}`

**Comment by scolladon** [comment]
Thanks @gavignon !

We'll look into it, it may introduce breaking changes but it will be closer to the standard behaviro

**Comment by mehdisfdc** [comment]
I agree that we were to launch the plugin today we should have implemented the behaviour described by @gavignon .

But since SGD is already heavily used, I have mixed feeling about introducing a breaking change. 

@gavignon are you facing an issue that would be solved by this change? 
Or did you log this issue simply to recommend that SGD follows the "usual" behaviour of the other `sfdx` commands?

**Comment by gavignon** [comment]
Thanks for the quick feedback @mehdisfdc @scolladon !

This is not a show stopper, I would like this enhancement to increase the readability in the jobs, as this output is not needed.
To avoid introducing breaking changes, maybe this can be done through a new parameter "--silent" ?

**Comment by mehdisfdc** [comment]
I like the idea of a new `--silent` flag

**Comment by scolladon** [comment]
This Enhancement as been "delivered" with the v6 version!

Now the plugin is compliant with the `--json` parameter behavior.
Without `--json` it displays a loader and the informations related to the plugin execution
With `--json` it displays a json information compliant with the `sf cli` spec
